nss-copper-dragonflies/BangazonAPI
# Jr product type

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Product Type Resource
- Can perform GET, PUT, POST, and DELETE
- view all product types available in the database 

Fixes # (issue)
#5 
## Type of change
Completed the ticket
Please delete options that are not relevant.


# Testing Instructions
1. `git fetch --all`
2. `git checkout jr-ProductType`
3. Run all integration tests make sure they are passing
4. Open post man test Get, Post, Put, and Delete

# Checklist:
